From 44662b9d8f19b3da146303afc1d573fef96788d5 Mon Sep 17 00:00:00 2001
From: Guy Nisan <gnisan@habana.ai>
Date: Wed, 02 Dec 2020 22:54:20 +0200
Subject: [PATCH] [SW-1833] preboot: fix version extraction

preboot.h is used for version extraction during build process. Using an
include from specs leads to an error. Removing include.

Change-Id: Ibb34d8713b60ae035fb7f4f6713ead1be70baf8a
Signed-off-by: Guy Nisan <gnisan@habana.ai>
---

diff --git a/include/preboot.h b/include/preboot.h
index ce14c13..3c043b6 100644
--- a/include/preboot.h
+++ b/include/preboot.h
@@ -36,7 +36,6 @@
 
 #include <stddef.h>
 #include <stdbool.h>
-#include <hl_fw_if.h>
 #include "auto_generated.h"
 
 #define VERSION_MAX_LEN		128
@@ -144,6 +143,8 @@
 bool preboot_sram_is_valid_address_range(u64 addr, int len);
 bool preboot_is_valid_address_range(u64 addr, int len);
 
+enum hl_fw_en_caps;
+
 void preboot_dev_enable_caps(void);
 void preboot_set_enabled_cap(enum hl_fw_en_caps en_cap);
 void preboot_finalize_enabled_caps(void);
